src/overlays/actors/ovl_En_Test3/z_en_test3.c

```diff
@@ -589,14 +589,14 @@ s32 func_80A3F384(EnTest3* this, PlayState* play) {
     EnDoor* door = (EnDoor*)func_80A3F2BC(play, this, ACTOR_EN_DOOR, ACTORCAT_DOOR, 55.0f, 20.0f);
     Vec3f offset;
 
-    if ((door != NULL) && (door->unk_1A1 == 0) &&
-        ((player->doorType == 0) || (&door->dyna.actor != player->doorActor)) &&
-        Actor_ActorAIsFacingActorB(&this->player.actor, &door->dyna.actor, 0x3000)) {
-        Actor_OffsetOfPointInActorCoords(&door->dyna.actor, &offset, &this->player.actor.world.pos);
+    if ((door != NULL) && (door->door.playOpenAnim == 0) &&
```

Comment:
```suggestion
    if ((door != NULL) && (!door->door.playOpenAnim) &&
```

Reply:
```suggestion
    if ((door != NULL) && !door->door.playOpenAnim &&
```
Without brackets too